src/draw/nema_gfx/lv_draw_nema_gfx_img.c

```diff
@@ -140,7 +140,9 @@ static void _draw_nema_gfx_img(lv_draw_unit_t * draw_unit, const lv_draw_image_d
     }
 
     uint32_t src_nema_cf = lv_nemagfx_cf_to_nema(src_cf);
-    uint32_t src_stride = img_dsc->header.stride;
+    /* the stride should be computed internally for NEMA_TSC images and images missing a stride value */
+    uint32_t src_stride = (src_cf >= LV_COLOR_FORMAT_NEMA_TSC_START && src_cf <= LV_COLOR_FORMAT_NEMA_TSC_END)
```

Comment:
Can we refine stride definition to better handle images that don't have stride?

Reply:
What did you have in mind?

Reply:
We can always calculate stride for TSC images either internally in NemaGFX or we can add the respective formats in lv_color_format_get_bpp to calculate it in LVGL. I will create a PR after we merge #7072

Reply:
I saw you use -1 for NEMA image format, is it appropriate to make it a macro say LV_STRIDE_INVALID?
If so, we may need to process INVALID in lv_draw_buf.c, lv_bin_decoder.c etc.

The vector image doesn't have stride either.

Reply:
Hmm I think it could be done. Although, all the draw units handle stride == 0 as a special meaning of `STRIDE_AUTO`. I think by adding `LV_STRIDE_INVALID` it creates more invalid states.

For the vector images, is stride ignored if it's been identified as a vector image? That's what's done here.

I could see a case for `LV_STRIDE_AUTO` only if TSC images could have either an auto stride or a separate stride but that doesn't seem to be the case.